table.go

```diff
@@ -3,13 +3,14 @@ package stickers
 import (
 	"errors"
 	"fmt"
-	"github.com/charmbracelet/lipgloss"
 	"log"
 	"math"
 	"reflect"
 	"strconv"
 	"strings"
 	"unicode/utf8"
+
+	"github.com/charmbracelet/lipgloss"
```

Comment:
please avoid these kind of IDE formatting changes

Reply:
Reverted in [b26919d](https://github.com/76creates/stickers/pull/5/commits/b26919dcf53b9c16472021aa1ca2679c71998240)